File: Week5/Day3/Solutions/Python/prog2.py
num = int(input("Octal number:"))
base = 1
dec=0
while(num):
    dec += (num%10)*base
    num//=10
    base *= 8

ans = ""
while(dec):
    rem = dec%16
    dec //= 16
    if rem<10:
        ans += str(rem)
    if rem>=10:
        ans += chr(rem+55)

print("Hexadecimal:",ans[::-1])

File: Week5/Day2/Solutions/C/prog2.c
int printf(const char*,...);
int scanf(const char*,...);

void main() {
	int num,rem=0,base=1;
	printf("Enter Octal number:");
	scanf("%d",&num);
	while(num){
		rem += (num%10)*base;
		num /= 10;
		base *= 8;
	}
	printf("Decimal number:%d\n",rem);
}
